operator/gitops/argocd/pipeline-service/pipelines-as-code/kustomization.yaml

```diff
@@ -20,3 +20,30 @@ patchesJson6902:
         path: "/metadata/labels"
         value:
           argocd.argoproj.io/managed-by: openshift-gitops
+  - target:
+      kind: Deployment
+      name: pipelines-as-code-controller
+      namespace: pipelines-as-code
+    patch: |-
+      - op: replace
+        path: "/spec/template/spec/containers/0/image"
+        value:
+          registry.redhat.io/openshift-pipelines/pipelines-pipelines-as-code-rhel8:v1.9.2-1
```

Comment:
Image changes do not require a patch of any kind. Use the `images` Kustomization object instead.

See https://github.com/openshift-pipelines/pipeline-service/blob/main/operator/gitops/argocd/pipeline-service/tekton-results/base/kustomization.yaml#L11-L20 for an example.